index.html

```diff
@@ -3,17 +3,17 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>TV Show Project | My Name (My GitHub username)</title>
+    <title>TV Show Project | illicitonion</title>
```

Comment:
I meant |  Daniel (illicitonion), i could be clearer

Reply:
Done